src/collections/_documentation/workflow/discover2/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Discover v2'
+sidebar_order: 4
+---
+
+{% capture __alert_content -%}
+The following is documentation for our new query builder Discover v2. Discover v1 will deprecate at the end of February 2020.
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Our Discover tool combines the best parts of our Events and existing Discover features, providing visibility into your data across environments and building upon and enriching your error data. This new feature will allow you to query and unlock deeper insights into the health of your entire system and answer critical business questions --- all in one place.
+
+[{% asset discover/discover-homepage.png alt="Full view of the Discover Homepage with query cards and button to build new queries." %}]({% asset discover/discover-homepage.png @path %})
+
+## Discover Homepage
+
+The Discover Homepage offers views of all your saved queries and Sentry's pre-built queries so you can quickly dive into areas that need immediate attention.
+
+### Query Cards
+
+Query cards are the individual cards on the Homepage that display a summarized view of the data in a specific query. Users can get a high-level view of the information in each query card to see what they want to investigate first. For example, if a spike occurred in the past 24 hours, a user may view that query first.
+
+### Pre-built Queries
+
+- Errors: Users can see their most frequent errors by total errors and unique errors. From the pre-built Error Query, users can also add additional columns and change the groupings to achieve their desired breakdown.
+- Project Summary: Users can see their most active projects --- by total Issues and unique Issues --- to see which projects need the most attention or further investigation.
+- Errors by URL: Users can see their most common errors broken down by specific URL.
+- Errors by Release: Users can query their errors by release to see if a change introduced in a particular release caused a spike in errors or to see if it fixed an issue.
+
```

Comment:
All Events: Users can see the raw error stream for any group of projects. This replaces the Events feature and gives users the ability to add additional columns and change groupings to achieve their desired breakdown.

Errors by Title: Users can see their most frequently occurring errors by total number of raw errors as well as total number of users that were affected.

Errors by URL: Users can see the pages that generated the most errors and different types of issues. As a reminder, Issues are a group of fingerprinted Events. (note: See here to learn more about [fingerprinting](https://docs.sentry.io/data-management/event-grouping/)).